automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/ProxyRedirectClient.scala

```diff
@@ -20,16 +19,18 @@ import scala.io.Source
 /**
  * Leonardo API service client.
  */
-object DummyClient extends RestClient with LazyLogging {
+object ProxyRedirectClient extends RestClient with LazyLogging {
 
   private val url = LeonardoConfig.Leonardo.apiUrl
 
-  def get(googleProject: GoogleProject, clusterName: RuntimeName)(implicit token: AuthToken,
-                                                                  webDriver: WebDriver): DummyClientPage = {
+  def get(googleProject: GoogleProject, clusterName: RuntimeName, tool: String)(
+    implicit webDriver: WebDriver,
+    token: AuthToken
+  ): ProxyRedirectPage = {
     val localhost = java.net.InetAddress.getLocalHost.getHostName
```

Comment:
Curious what `java.net.InetAddress.getLocalHost.getHostName` returns in a Jenkins environment.. and whether that works with Leo Referer validation.

Reply:
we may have to see what that returns per environment and add it to our valid referers in firecloud-develop